Guacamole/Assets/Scripts/MoleMovement.cs

```diff
@@ -0,0 +1,38 @@
+﻿ using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MoleMovement : MonoBehaviour
+{
+    public Transform pos1, pos2;
+    public float speed;
+    public Transform startPos;
+
+    Vector3 nextPos;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        nextPos = startPos.position; 
```

Comment:
It's not clear to me why this variable is called nextPos when it's just getting the position from startPos.

Reply:
I think it's because you need to assign initial point of nextPos we're using it for the movement later.